src/utils/defaultLanguageStrings.ts

```diff
@@ -142,7 +142,7 @@ export default {
   FILTER_ASSIGNER_ACTION: 'Assigner Action',
   FILTER_DATE_BEFORE: 'Before',
   FILTER_DATE_AFTER: 'After',
-  FILTER_CLEAR_ALL: 'Clear all filters',
+  FILTER_RESET_ALL: 'Reset all filters',
```

Comment:
Not part of this issue, but #1275 got merged before this suggestion was implemented.